boards/arc/nsim/support/mdb_hs_smp.args

```diff
@@ -23,6 +23,7 @@
 	-action_points=8
 	-Xstack_check
 	-interrupts=72
+	-firq
```

Comment:
How does that related to SMP fixes?

Reply:
The exception you found in nsim_hs_smp is caused by this.  mdb_hs_smp.arg is only for SMP.  so i think it can be part of SMP fixes